src/product.py

```diff
@@ -32,6 +36,26 @@ def product_info(serial_number):
     )
 
 
+@product_blueprint.route("/change_stock", methods=["POST", "GET"])
+@login_required
+@fields(request)
+def change_stock(SN, stock):

```

Comment:
Since we support type hints now in flask_require it would be nice with
```python
def change_stock(SN:int,stock:int)
```
or string if it's a string. What do you think?

Reply:
Yes, i agree. Fixed the problem.